app/controllers/jobs_controller.rb

```diff
@@ -43,7 +43,7 @@ def show
     if params[:flavor] == 'failed'
       job = Delayed::Job::Failed.find(params[:id])
     else
-      job = Delayed::Job.find(params[:id])
+      job = Delayed::Job::Failed.find(params[:id]) rescue Delayed::Job.find(params[:id])
```

Comment:
ewww rescue.  instead you can do

`Delayed::Job::Failed.find(Array.wrap(params[:id])).first ||
 Delayed::Job.find(params[:id])`

by passing an array, it will return an empty array if it's not found

I'm also concerned about this change in general, since regular jobs and failed jobs in the AR backend occupy different id namespaces (i.e you can have a job with id 1, and a failed job with id 1; this would effectively hide the non-failed one).